components/DetailsHeader/DetailsHeader.vue

```diff
@@ -51,21 +57,49 @@ export default {
 
   methods: {
     /**
-     * Formats title length
+     * Formats breadcrumb length
+     * @param {String} breadcrumb
+     */
+    formatBreadcrumb: function(breadcrumb) {
+      return breadcrumb.length > 32
+        ? breadcrumb.substring(0, 32) + '...'
+        : breadcrumb
+    },
```

Comment:
I think this is unused now that we have a breadcrumb component and can be removed.

Reply:
@matthewtgilbride We have a breadcrumb component? I wasn't aware lol let me check it out